knowledge_repo/app/templates/index-base.html

```diff
@@ -11,13 +11,46 @@
   {% endfor %}
 {% endmacro %}
 
+{% macro pagination(max_pages=10, extremes=True, autohide=True) %}
+  {% if feed_params %}
+    {% set start = feed_params['start'] | default(0) %}
+    {% set results = feed_params['results'] | default(10) %}
+    {% set page = 1 if (start == 0) else (start//results + 1) %}
+    {% set page_count = feed_params['page_count'] | default(1) %}
+
+    {% if autohide and page_count > 1 %}
+      {% set page_nums = pagination_pages(current_page=page, page_count=page_count, max_pages=max_pages, extremes=extremes) %}
+
+      <div class='pagination-bar' role="group">
+        <a href="{{ "javascript:;" if (page == 1) else modify_query(start=(page-2)*results) }}" class="pagination-stepper {{ "disabled" if page == 1 }}">
```

Comment:
Oh nice... I didn't release that `onclick='return false'` prevents the default event bubbling. That is a better way to do it.

It would have been cleaner to do something like:
```
<a href="{{ "#" if (page == 1) else modify_query(start=(page-2)*results) }}" class="pagination-stepper {{ "disabled" if page == 1 }}" {{ 'onclick="return false"' if page == 1 }}>
```